Makefile

```
@@ -204,8 +204,23 @@ endef
 
 ##@ oadp-nac specifics
 
+EC ?= $(LOCALBIN)/ec-$(EC_VERSION)
+EC_VERSION ?= 2.8.0
+
+.PHONY: editorconfig
+editorconfig: $(LOCALBIN) ## Download editorconfig locally if necessary.
+	@[ -f $(EC) ] || { \
+	set -e ;\
+	ec_binary=ec-$(shell go env GOOS)-$(shell go env GOARCH) ;\
+	ec_tar=$(LOCALBIN)/$${ec_binary}.tar.gz ;\
+	curl -sSLo $${ec_tar} https://github.com/editorconfig-checker/editorconfig-checker/releases/download/$(EC_VERSION)/$${ec_binary}.tar.gz ;\
+	tar xzf $${ec_tar} ;\
+	rm -rf $${ec_tar} ;\
+	mv $(LOCALBIN)/$${ec_binary} $(EC) ;\
+	}
+
 .PHONY: ci
-ci: simulation-test lint docker-build hadolint check-generate check-manifests ## Run all checks run by the project continuous integration (CI) locally.
+ci: simulation-test lint docker-build hadolint check-generate check-manifests ec ## Run all checks run by the project continuous integration (CI) locally.
```

> go fmt (run by golangci-lint) will take care of this for us in go files

move lint to after ec
```suggestion
ci: simulation-test docker-build hadolint check-generate check-manifests ec lint ## Run all checks run by the project continuous integration (CI) locally.
```

---

`make ec` will only read

the vscode plugin and other stuff do the writing part, but did not find an executable, so we wont have `make ec-fix` command :(